MapboxNavigation/RouteVoiceController.swift

```diff
@@ -61,6 +54,24 @@ public class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate {
     
     
     /**
+     If true, a noise indicating the user is going to be rerouted will play prior to rerouting.
+     */
+    public var playRerouteSound = false
+
+    
+    /**
+     Sound to play prior to reroute. Inherits volume level from `volume`.
+     */
+    public var rerouteSound = NSDataAsset(name: "reroute-sound", bundle: Bundle.navigationUI)
```

Comment:
@frederoni how do you feel about the type here? Is there a way we could make this API easier to use?